src/Cake.Tfs/TfsAliases.PullRequest.cs

```diff
@@ -55,6 +55,34 @@ public static partial class TfsAliases
             return null;
         }
 
+        /// <summary>
+        /// Gets a Team Foundation Server or Azure DevOps pull request using the settings obtained from predefined build variables.
+        /// </summary>
+        /// <param name="context">The context.</param>
+        /// <example>
+        /// <para>Get a pull request:</para>
+        /// <code>
+        /// <![CDATA[
+        ///     var pullRequest = TfsPullRequestUsingTfsBuildOAuthToken();
+        /// ]]>
+        /// </code>
+        /// </example>
+        /// <returns>Description of the pull request.
+        /// Returns null if pull request could not be found and
+        /// <see cref="TfsPullRequestSettings.ThrowExceptionIfPullRequestCouldNotBeFound"/> is set to <c>false</c>.</returns>
+        /// <exception cref="TfsPullRequestNotFoundException">If pull request could not be found and
+        /// <see cref="TfsPullRequestSettings.ThrowExceptionIfPullRequestCouldNotBeFound"/> is set to <c>true</c>.</exception>
```

Comment:
We should also document the possible exceptions which can occur if not running on Azure Pipelines or if environment variables are not set (e.g. because option to provide OAuth token to scripts is not enabled).